src/gui/plugins/view_angle/ViewAngle.qml

```diff
@@ -391,6 +391,41 @@ ColumnLayout {
     }
   }
 
+  // Set camera's FOV
+  Text {
+    text: "Camera FOV"
+    Layout.fillWidth: true
+    color: Material.Grey
+    leftPadding: 5
+    topPadding: 10
+    font.bold: true
+  }
+
+  GridLayout {
+    width: parent.width
+    columns: 4
+
+    Text {
+      text: "horizontal fov (degree)"
```

Comment:
For the UI, it's always a good idea to capitalize the words

```suggestion
      text: "Horizontal FOV (degree)"
```

Reply:
Changed in f7592a3